lib/setup.js

```diff
@@ -215,12 +215,13 @@ export default function (app) {
   })
 
   // Error handler at the end
-  const dir = path.resolve(__dirname, '../desktop/components/error_handler')
-  app.set('view engine', 'jade')
-  app.set('views', dir)
+  const file = path.resolve(
```

Comment:
I noticed `Error: No default engine was specified and no extension was provided. ` in the logs pointing here. It looks like we can simply give the full path to the jade file instead of having to specify `view engine` or `views`.